### PR TITLE
Leetcode-solutions

### PR DESCRIPTION
**Pull request by kashif-raza2019**
Uploaded Files after so long!

### CODE REVIEW

**Comment by kashif-raza2019** [comment]
Some conflicts, i think due to existing files ...

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      "intelliSenseMode": "windows-gcc-x86",
+      "compilerArgs": [
+        "-Wall",
+        "-Wextra",
+        "-Wpedantic"
+      ]
+    }
+  ],
+  "version": 4
+}
```
Comment (AdityaPrakash-26) [inline .vscode/c_cpp_properties.json]: Please undo this change :)

**Review by AdityaPrakash-26 — CHANGES_REQUESTED** [review]

File: .vscode/c_cpp_properties.json (hunk 1 above)
Comment (kashif-raza2019) [reply .vscode/c_cpp_properties.json]: Okay

**Comment by AdityaPrakash-26** [comment]
I've resolved the conflicts :) thanks for the PR!

**Pull request merged**